docs/css/ez-guidelines-buttons.css

```diff
@@ -15,6 +15,10 @@
     font-size: 1em;
     color: #fff;
 }
+.ez-guidelines .ez-code-example .btn:disabled {
```

Comment:
What do you think about adding also: `.ez-guidelines .ez-code-example button[disabled]` selector?

Reply:
Updated based on review.